src/views/Home/NewPage.js

```diff
@@ -0,0 +1,45 @@
+import { useState } from 'react';
+import DogForm from '../Admin/DogForm';
+import { createDog } from '../../services/dogs';
+
+export default function NewPage() {
+  const [name, setName] = useState('');
+  const [age, setAge] = useState('');
+  const [bio, setBio] = useState('');
+  const [breed, setBreed] = useState('');
+  const [image, setImage] = useState('');
```

Comment:
As a refactor, we could utilize one state object. Definitely not a requirement though.